setup.py

```diff
@@ -1,21 +1,15 @@
-import setuptools
+from setuptools import setup
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
-
-setuptools.setup(
-    name="kaisdk",
-    version="0.0.15",
-    author="Vicara",
-    author_email="dev@vicara.co",
-    description="A python package to assist in interfacig with the KAI SDK",
-    long_description=long_description,
-    long_description_content_type="text/markdown",
-    url="https://github.com/vicara-hq/kai-python",
-    packages=setuptools.find_packages(),
-    classifiers=[
-        "Programming Language :: Python :: 3",
-        "License :: OSI Approved :: MIT License",
-        "Operating System :: OS Independent",
+setup(
+    name='kai-python',
```

Comment:
Is this the name of the package that'll be published on pip? If so, we should have two different packages, one for process module and one for websocket module

Reply:
Yes, that's the name published on pip. Actually, I was thinking we can include the process module code in this library itself, the only thing that changes is the implementation of the send() method. And the people can use the same library to build both process modules and websocket modules, by importing the respective class:

`from KaiSDK.WebSocketModule import WebSocketModule`
or
`from KaiSDK.ProcessModule import ProcessModule`